plugins/inputs/fritzbox/README.md

```diff
@@ -0,0 +1,179 @@
+    - `fritz_host` - The host connected to the network
+    - `fritz_host_role` - The host's role ("master" = mesh master, "slave" = mesh slave, "client") in the network
+    - `fritz_host_peer` - The name of the peer this host is connected to
+    - `fritz_host_peer_role` - The peer's role ("master" = mesh master, "slave" = mesh slave, never "client") in the network
+    - `fritz_link_type` - The link type ("WLAN" or "LAN") of the peer connection
+    - `fritz_link_name` - The link name of the connection
+  - fields
+    - `max_data_rate_tx` (uint) - The connection's maximum transmit rate (kilobits/sec)
+    - `max_data_rate_rx` (uint) - The connection's maximum receive rate (kilobits/sec)
+    - `cur_data_rate_tx` (uint) - The connection's maximum transmit rate (kilobits/sec)
+    - `cur_data_rate_rx` (uint) - The connection's current receive rate (kilobits/sec)
+
+## Example Output
+
+<!-- markdownlint-disable MD013 -->
+
+```text
+fritzbox_device,fritz_device=127.0.0.1,fritz_service=DeviceInfo1 model_name=Mock 1234,serial_number=123456789,hardware_version=Mock 1234,software_version=1.02.03,uptime=2058438 1736529975
+
+fritzbox_wan,fritz_device=127.0.0.1,fritz_service=WANCommonInterfaceConfig1 total_bytes_received=554484531337,layer1_upstream_max_bit_rate=48816000,layer1_downstream_max_bit_rate=253247000,upstream_current_max_speed=511831,downstream_current_max_speed=1304268,total_bytes_sent=129497283207 1736530024
+
+fritzbox_ppp,fritz_device=127.0.0.1,fritz_service=WANPPPConnection1 uptime=369434,upstream_max_bit_rate=44213433,downstream_max_bit_rate=68038668 1736530058
+
+fritzbox_dsl,fritz_device=127.0.0.1,fritz_service=WANDSLInterfaceConfig1 downstream_noise_margin=60,upstream_power=498,downstream_power=513,upstream_curr_rate=46719,downstream_curr_rate=249065,upstream_max_rate=48873,downstream_max_rate=249065,upstream_noise_margin=80,severly_errored_secs=0,upstream_attenuation=80,transmit_blocks=254577751,init_timeouts=0,atuc_crc_errors=13,receive_blocks=490282831,errored_secs=25,fec_errors=0,atuc_hec_errors=0,atuc_fec_errors=0,hec_errors=0,crc_errors=53,downstream_attenuation=140,cell_delin=0,link_retrain=2,init_errors=0,loss_of_framing=0 1736530092
+
+fritzbox_wlan,fritz_device=127.0.0.1,fritz_service=WLANConfiguration1,fritz_wlan=MOCK1234,fritz_wlan_channel=13,fritz_wlan_band=2400 total_associations=11 1736530130
+
+fritzbox_host,fritz_host_peer_role=master,fritz_link_type=WLAN,fritz_link_name=AP:2G:0,fritz_device=127.0.0.1,fritz_service=Hosts1,fritz_host=device#17,fritz_host_role=slave,fritz_host_peer=device#1 max_data_rate_tx=216000,max_data_rate_rx=216000,cur_data_rate_tx=216000,cur_data_rate_rx=216000 1736530165
+fritzbox_host,fritz_device=127.0.0.1,fritz_service=Hosts1,fritz_host=device#24,fritz_host_role=client,fritz_host_peer=device#17,fritz_host_peer_role=slave,fritz_link_type=LAN,fritz_link_name=LAN:1 cur_data_rate_tx=0,cur_data_rate_rx=0,max_data_rate_tx=1000000,max_data_rate_rx=1000000 1736530165
```

Comment:
These are not valid line protocol metrics.

Reply:
Indeed, I will rework this.

Reply:
Done via fd007fccfb4d6a2e9825ed61f5b702dc2346c2ca